src/pretix/static/pretixcontrol/scss/main.scss

```
@@ -841,9 +841,20 @@ h1 .label {
 tbody[data-dnd-url] {
     transition: opacity 1s;
 }
+.table-items tbody + tbody {
+  border-top-width: 1px;
+}
+.table-items tbody[data-dnd-url]:after {
+    content:'';
+    display: table-row;
+    height: 0.5em;
+}
 .sortable-sorting tbody:not(.sortable-dragarea) {
     opacity: .4;
 }
+th span.not-bold {
```

Maybe a more generic `.font-normal` with a font-weight and -style: normal would be more practical? But feel free to leave it as-is.

```suggestion
.font-normal {
    font-style: normal;
```